src/test/java/uk/gov/justice/digital/delius/transformers/OffenderTransformerTest.java

```diff
@@ -50,7 +54,7 @@ public void disabilitiesCopiedWithLatestFirst() {
                                 .disabilityType(StandardReference
                                         .builder()
                                         .codeValue("A")
-                                        .codeDescription("Metal health")
```

Comment:
\m/

Reply:
The base data actually sets this up and then checks for it. I had a note to check this with you!